src/spinner.styles.ts

```diff
@@ -0,0 +1,56 @@
+import { css } from 'lit';
+
+export default [
+  css`
+    @keyframes spin {
+      100% {
+        align-items: center;
+        transform: rotate(1turn);
+      }
+    }
+
+    .component {
+      background:
+        radial-gradient(
+            farthest-side,
+            var(--glide-core-color-interactive-icon-active) 94%,
+            #0000
```

Comment:
To hard code or not to hard code? Feels like a one-off to me.

Reply:
Not sure if Zheng would have any suggestions for an existing variable, but otherwise I think this is okay.

Reply:
Sounds good to me. We can always revisit if needed.